content/guides/figma/getting-started.mdx

```diff
@@ -18,30 +19,81 @@ You can also directly open the *team library* view via the command palette or wi
   />
 </ImageBox>
 
-<ImageBox caption="Enable all libraries you need in your file.">
+<ImageBox caption="Enable 'Primer Web' and all other libraries you need in your file.">
   <img
-    width="443"
+    width="508"
     alt="Screenshot showing the library overview in figma"
-    src="https://user-images.githubusercontent.com/813754/221527802-76638b6b-be81-4c87-b3e8-63d543537a9c.png"
+    src="https://github.com/primer/react/assets/813754/675eb97e-479d-47db-bd29-f10c136f586a"
   />
 </ImageBox>
 
+### Styles and variables
 
-## Understanding styles
-Figma libraries like `Primer Primitives` install styles for you to use. In contrast to *local styles* they don't show up in the sidebar.
-However once you open a selection to choose a color style, text style, etc. you will see the styles from team libraries as well.
+Styles and variables are two very similar ways of bringing `design tokens` like [Primer Primitives](https://github.com/primer/primitives) into Figma.
+Our goal is to move everything over to variables, but at the moment only color and size tokens are supported. This is why we still provide text and shadow tokens using styles.
 
-To quickly find a style you can use the search box. E.g. use `accent` to bring up all accent colors.
+### Understanding styles
+Figma libraries like `Primer Web` provide styles for you to use. In contrast to local styles, styles from a library don't show up in the sidebar. However once you open a selection to choose a `text style` or `shadow` you will see the styles from team libraries as well.
```

Comment:
Non-blocking nitpick: I'm not sure we want "Primer Web", "text style", or "shadow" in a `<code>` tag. I think this is intended for things like variable names.

Reply:
Hey, so for Primer Web, yes.

For text and shadow styles I am wondering if this is correct as they are specific Figma things. Similar to a variable or a property. WDYT?

Reply:
I guess that's ok. I personally wouldn't do it that way, but I don't think anybody will be confused 🙂